core/src/main/resources/jenkins/monitor/JavaLevelAdminMonitor/message.properties

```diff
@@ -0,0 +1,3 @@
+Recommended_Java_Version=Java 11 is the recommended version to run Jenkins on, please consider upgrading. \
+  Later in 2021 Jenkins will drop support for running the controller on Java 8. Your builds can still be run on \
```

Comment:
Are we sure we will be dropping it in 2021? Perhaps when we have an appropriate timeline this should be updated?